### ISSUE TITLE
Detecting TP issues in RL (token logprob error is not sufficient)

### ISSUE THREAD

**Issue by terrykong** [NVIDIA-NeMo/RL#1288]
In an acemath experiment, it was discovered that the TP plan in the DtensorWorkerPolicy (V1) has a negative affect on convergence. It was root-caused to rowwise parallel parallelization for TP, but the fix is not known yet other than not using rowwise parallel and this is related to #1227 which also shows that V2 has this issue as well.

We always assumed that log prob error was sufficient to diagnose divergence, but this exercise has shown that it isn't. We need to formulate a new metric/criteria for accepting that a model may be problematic. 

Some options include:
1. histograms of token prob errors (to look at distributions instead of summary scalars)
2. differences in logits

(1) may not be sensitive enough and (2) may be too sensitive, so we need to find a metric that is indicative enough that there is a problem.

The experimentation should be the acemath experiment since that is the only place we've confirmed there's an issue

**Comment by qiaochuz-nv** [comment]
## QA RCCA Analysis

RCCA Status: Not Required
RCCA Category: not_required
Test Action: none

Fix PR: N/A
Root Cause: This issue is not an actionable closed product-code defect for RCCA; it is enhancement/feature tracking, docs/cosmetic work, placeholder/test issue, or closed non-bug work.
Trigger Config: N/A.

Coverage Assessment:
| Area | Result |
|---|---|
| Existing Test | N/A |
| Coverage Status | NOT_REQUIRED |
| Gap | N/A |
| Corrective Action | No RCCA regression test required. |

Conclusion: RCCA not required because this does not represent a QA coverage escape for a fixed NeMo-RL bug.
